app/assets/stylesheets/tamu.scss

```diff
@@ -0,0 +1,71 @@
+#header-navbar .navbar-brand {
+  background: transparent "tamu-logo-with-bar.png" no-repeat top left;
+  margin-top: 10px;
+  margin-bottom: -10px;
+}
+
+#header-navbar .navbar-toggle, #header-navbar .facets-toggle {
+  background: #ffcf08;
+}
+
+.container > .navbar-header, .container > .navbar-collapse, .container > #facet-panel-collapse, .container > div[id*='facet-panel-'], .container-fluid > .navbar-header, .container-fluid > .navbar-collapse, .container-fluid > #facet-panel-collapse, .container-fluid > div[id*='facet-panel-'] {
+  background-color: #500000;
```

Comment:
Can we update this scss file to import the bootstrap theme variables and use them where applicable? Also, we should be able to remove overrides of bootstrap classes in which the same thing the variable would apply by default.